core/commands.py

```diff
@@ -206,7 +206,13 @@ def finish_process(self):
             self.output_file.close()
             progress = MinioProgress()
             self.log(f"Uploading file to bucket: {bucket}")
-            self.minio.fput_object(bucket, dest_name, self.output_file.name, progress=progress)
+            if suffix == ".csv.gz":
+                content_type = "application/gzip"
+            elif suffix == ".csv":
+                content_type = "text/csv"
```

Comment:
Nem todos os arquivos que vamos hospedar serão somente csv/csv.gz. No mirror (que num futuro próximo usará esse código) teremos arquivos zip e outros formatos também. Nesse caso, seria legal usar `mimetypes.guess_type`, algo assim:

```suggestion
            import mimetypes
            content_type, encoding = mimetypes.guess_type(dest_name)
            if encoding == "gzip":  
                # quando é '.csv.gz' o retorno de guess_type é ('text/csv', 'gzip')
                content_type = "application/gzip"
```

> Nota: em alguns casos essa função pode devolver `(None, None)` e talvez precisemos tratar, como em `mimetypes.guess_type("SHA512SUMS")`.

Reply:
Resolvido em de95f49